content/blog/diegoroux/2024-07-27_[gsoc_2024]_recording_achieved_more.md

```diff
@@ -0,0 +1,77 @@
++++
+type = "blog"
+title = "[GSoC 2024] Virtio Sound: Recording Achieved and more"
+date = "2024-07-27 00:41:37-06:00"
+tags = ["gsoc", "gsoc2024", "virtio", "sound"]
++++
+
+# intro
+Hello, once again! It's been weeks since the last update, so here we go.
+
+Good news, there is active development of the virtio sound driver for Haiku.
+Key progress includes recording on the device and improvements. 
+
+
+# buffer exchange redesign
+As per the last blog post:
+
+We still specify 2 playback buffers, but incremented to 16,384 frames per
```

Comment:
"incremented": or extended?

Reply:
What I meant was the 'increment' of FRAMES_PER_BUFFER